webpack.config.js

```diff
@@ -208,6 +208,10 @@ if ( calypsoEnv === 'production' ) {
 	) );
 }
 
+if ( ! config.isEnabled( 'desktop' ) ) {
+	webpackConfig.plugins.push( new webpack.NormalModuleReplacementPlugin( /^lib\/desktop$/, 'lodash/noop' ) );
```

Comment:
In cases where we handle files like so, whether that be with a replacement, a custom loader or something else a little unorthodox, might it be worth documenting this somewhere within that module?

I've found myself quite confused with one of the server-side modules while tinkering with the desktop merge and eventually found that a custom loader was causing unexpected (at least for me) behaviour.

Out of the scope of this PR of course - but I'm interested to hear your thoughts!